frontend/app/src/components/home/Home.svelte

```diff
@@ -690,6 +690,10 @@
         });
     }
 
+    function freezeGroup(ev: CustomEvent<string>) {
+        client.freezeGroup(ev.detail);
```

Comment:
We might be swallowing any error here. Typical pattern is for the _client_ to simply resolve to a `Promise<boolean>` based on overall success / failure. Component code will then toast an error message if it fails. Internally the client lib will log the actual api response and the exact failure mode. Should be able to find plenty of other similar examples I think.